src/rpn2_geoclaw.f

```diff
@@ -57,14 +59,22 @@ subroutine rpn2(ixy,maxm,meqn,mwaves,maux,mbc,mx,
       double precision fw(3,3)
       double precision sw(3)
 
-      double precision hR,hL,huR,huL,uR,uL,hvR,hvL,vR,vL,phiR,phiL
+      double precision hR,hL,huR,huL,uR,uL,hvR,hvL,vR,vL,phiR,phiL,pL,pR
       double precision bR,bL,sL,sR,sRoe1,sRoe2,sE1,sE2,uhat,chat
       double precision s1m,s2m
       double precision hstar,hstartest,hstarHLL,sLtest,sRtest
       double precision tw,dxdc
 
       logical rare1,rare2
 
+      ! Density used in pressure gradient calculation 
+      ! TODO - Should add this as a parameter to geoclaw_module
```

Comment:
I would also probably add the density of air given that comes up every once in awhile as well.